specification/standbypool/resource-manager/Microsoft.StandbyPool/preview/2024-03-01-preview/standbypool.json

```diff
@@ -1535,7 +1535,7 @@
       "properties": {
         "instanceCountSummary": {
           "type": "array",
-          "description": "A list containing the counts of virtual machines in each possible power state for each zone if enabled, as known by the StandbyPool resource provider.\nIf zones are not enabled on the attached VMSS, the list will contain a single entry with null zone values.\nNote: any updates to pool resources outside of StandbyPoolRP (i.e deleting a VM through portal) are not reflected here.\nNote: any resources in the Running state may still be installing extensions / not fully provisioned.",
+          "description": "A list containing the counts of virtual machines in each possible power state for each zone if enabled, as known by the StandbyPool resource provider.\nIf zones are not enabled on the attached VMSS, the list will contain a single entry without zone values.\nNote: any resources in the Running state may still be installing extensions / not fully provisioned.",
```

Comment:
>\n [](http://example.com/codeflow?start=176&length=2)

why are these \n needed . here and other places of this PR

Reply:
Thanks for pointing out, fixed.